packages/guui/dropdown/index.js

```diff
@@ -20,131 +21,139 @@ type Props = {
-            verticalAlign: 'middle',
-            transition: 'transform 250ms ease-out',
-        },
-    },
-
-    input: {
-        ...screenReaderOnly,
-        ':checked + ul': {
-            display: 'block',
-        },
-    },
-
-    'button.expanded:after, label.expanded:after': {
-        transform: 'translateY(1px) rotate(-135deg)',
-    },
-
-    ul: {
-        zIndex: 1072,
-        listStyle: 'none',
-        backgroundColor: 'white',
-        padding: '6px 0',
-        boxShadow: '0 0 0 1px rgba(0,0,0,0.1)',
-        borderRadius: '3px',
-        position: 'absolute',
-        right: '0',
-        width: '200px',
-        display: 'none',
-    },
-
-    'ul.expanded': {
-        display: 'block',
-    },
-
-    'li a': {
-        display: 'block',
-        padding: '10px 18px 15px 30px',
-    },
-
-    a: {
-        fontSize: '15px',
-        fontFamily: textSans,
-        color: palette.neutral['1'],
-        lineHeight: 1.2,
-        position: 'relative',
-        transition: 'color 80ms ease-out',
-        padding: '6px 10px',
-        margin: '-1px 0 0 0',
-        textDecoration: 'none',
-        ':hover': {
-            textDecoration: 'underline',
-        },
-        ':focus': {
-            textDecoration: 'underline',
-        },
-
-        ':before': {
-            content: '""',
-            borderTop: '1px solid #ededed',
-            display: 'block',
-            position: 'absolute',
-            top: 0,
-            left: '30px',
-            right: 0,
-        },
-    },
-
-    // no top border for first item
-    'li:first-child a:before': {
-        content: 'none',
-    },
-
-    'a:hover': {
-        backgroundColor: '#ededed',
-        textDecoration: 'none',
-    },
-
-    'a.active': {
-        fontWeight: 'bold',
-
-        ':after': {
-            content: '""',
-            border: '2px solid #c70000',
-            borderTop: 0,
-            borderRight: 0,
-            position: 'absolute',
-            top: '14px',
-            left: '10px',
-            width: '10px',
-            height: '4px',
-            transform: 'rotate(-45deg)',
-        },
-    },
-});
+const input = css`
+    // TODO re-add screen-reader only mixin
+    :checked + ul {
+        display: block;
+    }
+`;
+
+const ul = css`
+    z-index: 1072;
+    list-style: none;
+    background-color: white;
+    padding: 6px 0;
+    box-shadow: 0 0 0 1px rgba(0, 0, 0, 0.1);
+    border-radius: 3px;
+    position: absolute;
+    right: 0;
+    width: 200px;
+    display: none;
+`;
+
+const ulExpanded = css`
+    ${ul};
+    display: block;
+`;
+
+const link = css`
+    font-size: 15px;
+    font-family: ${textSans};
+    color: ${palette.neutral['1']};
+    line-height: 1.2;
+    position: relative;
+    transition: color 80ms ease-out;
+    padding: 6px 10px;
+    margin: -1px 0 0 0;
+    text-decoration: none;
+
+    display: block;
+    padding: 10px 18px 15px 30px;
+
+    :hover {
+        text-decoration: underline;
```

Comment:
No need for this rule `text-decoration: underline` as it's being overridden 2 lines later with `text-decoration: none;`

Reply:
ah thanks!